CMakeLists.txt

```diff
@@ -170,6 +170,7 @@ target_compile_definitions(qjs PRIVATE ${qjs_defines})
 if (CMAKE_BUILD_TYPE MATCHES Debug OR DUMP_LEAKS)
     target_compile_definitions(qjs PRIVATE
         DUMP_LEAKS
+        DUMP_LEAKS_FLAG=0
```

Comment:
Wouldn't this disable it?

Reply:
setting `DUMP_LEAKS_FLAG` to `0` makes the output unconditional. Any other value makes the output selectable with the `-d<bitmask>` command line option (or a future javascript extended command).

Reply:
This conversation is resolved as a more recent commit gives `DUMP_LEAKS` its previous behavior.